definitions/procedures/content/switchover.rb

```diff
@@ -0,0 +1,18 @@
+module Procedures::Content
+  class Switchover < ForemanMaintain::Procedure
+    metadata do
+      description 'Convert supported content from Pulp 2 to Pulp 3'
```

Comment:
I'd actually change this to 'Switch support for certain content from Pulp 2 to Pulp 3'

Reply:
@jlsherrill updated!